### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by keith-merchant-ibm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
 # Getting Started
 
-This repo is a variety of code snippets to be used by developers to interact with resources that belong to VPC on Classic IaaS Offering. This repository is intended to be used for documentation purposes only and not to be included as a dependency.
-
-These examples are provided in the following languages.
+This repository consists of a variety of code snippets intended to aid developer interaction with resources belonging to VPC on classic laaS Offering. This repository is intended to be used for documentation purposes only and not to be included as a dependency.
```
Comment (astha-jain) [inline README.md]: It should be `VPC on Classic IaaS offering` as it is the name of offering.

File: README.md [hunk 2]
```diff
@@ -243,7 +254,7 @@ Following this section, you will be able to call VPC APIs in your Go workspace.
 
 ## Python
 
-The following steps gives an example on how to retrieve a token, list all VPCs, and create a VPC.
+The the example below shows how to retrieve a token, list all VPCs, and create a VPC.
```
Comment (astha-jain) [inline README.md]: here.

**Review by astha-jain — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (keith-merchant-ibm) [reply README.md]: Corrected

File: README.md (hunk 2 above)
Comment (keith-merchant-ibm) [reply README.md]: Corrected

**Review by astha-jain — APPROVED** [review]
Looks good 👍

**Pull request merged**